enterprise/frontend/src/embedding-sdk/components/public/MetabaseProvider.tsx

```diff
@@ -93,3 +97,23 @@ export const MetabaseProvider = memo(function MetabaseProvider(
 ) {
   return <MetabaseProviderInternal store={store} {...props} />;
 });
+
+const PluginsActivator = () => {
+  // it needs to be nested in the provider to be able to use useSelector
+  // we can't just use useSelector in the MetabaseProvider because it would be outside of  redux's <Provider>
+
+  const tokenFeatures = useSelector(state =>
+    getSetting(state, "token-features"),
+  );
+
+  useEffect(() => {
+    if (
+      tokenFeatures &&
+      Object.values(tokenFeatures).some(value => value === true)
```

Comment:
Maybe we just need to make sure they're loaded, I'm not sure if some plugins are activated even without any feature